data/helpers.d/package

```diff
@@ -1,10 +1,34 @@
+# Check if apt is free to use, or wait, until timeout.
+#
+# [internal]
+#
+# usage: ynh_is_dpkg_free
+ynh_is_dpkg_free() {
```

Comment:
Is it better to use a name like `ynh_wait_dpkg_free` to explain that it's not just a check.

Reply:
Honestly, I don't care about the name of the helper.

Reply:
Change done